app/views/donations/_donation_row.html.erb

```diff
@@ -4,7 +4,7 @@
   <td><%= donation_row.donation_site.try(:name) %></td>
   <td><%= donation_row.storage_location.name %></td>
   <td class="numeric"><%= donation_row.line_items.total %></td>
-  <td class="numeric"><%= donation_row.line_items.size %></td>
+  <td class="numeric"><%= item_value(donation_row.money_raised.to_i) %></td>
```

Comment:
Can you add a screenshot of what this looks like? I'm not sure if numeric is still the right class.

Reply:
I think this is correct.

<img width="609" alt="Screen Shot 2019-11-06 at 12 17 39 AM" src="https://user-images.githubusercontent.com/78151/68270322-ed6a1400-002a-11ea-9f7a-62bcd3baa13a.png">

Reply:
`numeric` is the correct CSS class here, yeah